src/wh_nvm_flash.c

```diff
@@ -512,16 +512,19 @@ static int nfPartition_CheckDataRange(whNvmFlashContext* context,
     return WH_ERROR_OK;
 }
 
-static uint32_t nfObject_Offset(whNvmFlashContext* context, int partition,
-        int object_index)
+static int nfObject_Offset(whNvmFlashContext* context, int partition,
+        int object_index, uint32_t *offset)
```

Comment:
nit: consider prefixing output pointers with `out`
```suggestion
        int object_index, uint32_t *out_offset)
```

Reply:
Done